pyquil/gates.py

```diff
@@ -445,16 +445,17 @@ def PSWAP(angle, q1, q2):
 
 def RESET(qubit_index=None):
     """
-    Reset all qubits or just a specific qubit at qubit_index.
+    Reset all qubits or just one specific qubit.
 
-    :param Optional[int] qubit_index: The address of the qubit to reset.
+    :param Optional[qubit_index] qubit: The qubit to reset. This can be a qubit's index, a Qubit,
```

Comment:
`qubit` -> `qubit_index`

Reply:
What should I put for the param type in brackets? Union[int, Qubit, QubitPlaceholder]?

Reply:
I think `integer_types` rather than `int`.

Reply:
Good call. Fixing now.